addons/mail/static/tests/mail_status_manager_tests.js

```diff
@@ -1,190 +0,0 @@
-odoo.define('mail.mailStatusServiceTests', function (require) {
-"use strict";
-
-var mailTestUtils = require('mail.testUtils');
-
-var core = require('web.core');
-var testUtils = require('web.test_utils');
-var Widget = require('web.Widget');
-
-var QWeb = core.qweb;
-
-QUnit.module('mail', {}, function () {
-QUnit.module('service', {}, function () {
-QUnit.module('Status manager', {
-    beforeEach: function () {
-        this.services = mailTestUtils.getMailServices(this);
-        this.timeoutMock = mailTestUtils.patchMailTimeouts();
```

Comment:
I think patch of mail timeouts is only used with fetch im status. If so, these test helpers should be removed as well

Reply:
I'll check that